KafkaIngester/src/main/java/io/deephaven/kafka/ingest/JsonNodeObjectFieldCopier.java

```diff
@@ -0,0 +1,24 @@
+package io.deephaven.kafka.ingest;
+
+import io.deephaven.db.v2.sources.chunk.Attributes;
+import io.deephaven.db.v2.sources.chunk.ObjectChunk;
+import io.deephaven.db.v2.sources.chunk.WritableObjectChunk;
+import io.deephaven.db.v2.sources.chunk.WritableChunk;
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class JsonNodeObjectFieldCopier implements FieldCopier {
+    private final String fieldName;
+
+    public JsonNodeObjectFieldCopier(String fieldName) {
+        this.fieldName = fieldName;
+    }
+
+    @Override
+    public void copyField(ObjectChunk<Object, Attributes.Values> inputChunk, WritableChunk<Attributes.Values> publisherChunk, int sourceOffset, int destOffset, int length) {
+        final WritableObjectChunk<Object, Attributes.Values> output = publisherChunk.asWritableObjectChunk();
+        for (int ii = 0; ii < length; ++ii) {
+            final JsonNode node = (JsonNode) inputChunk.get(ii + sourceOffset);
+            output.set(ii + destOffset, JsonNodeUtil.getValue(node, fieldName, true, true));
```

Comment:
What kind of objects do we expect here?

Reply:
We discussed over slack, we are deleting.